UPGRADE_GUIDE.md

```diff
@@ -1,5 +1,34 @@
 # Upgrade Guide
 
+## From `0.3.3` to `0.3.4`
+
+### PlayerLifecycle feature module now provides an EntityId
+
+The callback used for creating a player `EntityTemplate` has changed to provide an `EntityId` up front.
+This will be the `EntityId` the player's Entity will have after it is successfully spawned, and can be usefull for QBI.
```

Comment:
```suggestion
This player's Entity will have this `EntityId` after it is successfully spawned, and can be useful for defining QBI queries.
```

Reply:
not sure if we should mention "can be useful for QBI" if we don't give any examples of why it's useful

Reply:
I think that's a little out of scope maybe for this document. Thats where the QBI guide comes in right?

Reply:
Exactly, so let's not mention QBI until we have the guide